brim/zeekrunner

```diff
@@ -2,8 +2,8 @@
 
 dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
-export ZEEKPATH=$dir/share/zeek:$dir/share/zeek/policy:$dir/share/zeek/site
-export ZEEK_PLUGIN_PATH=$dir/lib/zeek/plugins
+export ZEEKPATH="$dir/share/zeek:$dir/share/zeek/policy:$dir/share/zeek/site"
+export ZEEK_PLUGIN_PATH="$dir/lib/zeek/plugins"
```

Comment:
This should work properly as is, even when dir contains spaces. Is it not for you?

Reply:
TIL. `shellcheck` does not complain about the LHS of this diff.